vendor/knative.dev/eventing/test/test_images/logevents/main.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
would it make sense to move this test images to pkg? maybe not because is too eventing specific?

Reply:
Yeah it is too eventing specific. Currently all the code under `pkg/test` are generic to all Knative projects.